aframe/tasks/data/timeslide_waveforms/timeslide_waveforms.py

```diff
@@ -198,11 +198,19 @@ def rejected_parameter_files(self):
         return list(map(Path, [targets[1].path for targets in self.targets]))
 
     def run(self):
-        from ledger.injections import InjectionParameterSet, LigoResponseSet
+        from ledger.injections import (
+            InjectionParameterSet,
+            InterferometerResponseSet,
+            waveform_class_factory,
+        )
 
-        LigoResponseSet.aggregate(
-            self.waveform_files, self.waveform_output, clean=True
+        cls = waveform_class_factory(
+            list(map(lambda x: x.lower(), self.ifos)),
```

Comment:
This line gets used a few times, so it might make sense to have it be in `waveform_class_factory`. Also, I haven't used this kind of functionality much, is this equivalent to `[x.lower() for x in self.ifos]`?

Reply:
yeah that's actually a good point I'll move it there. And yeah I think they're equivalent, the list comprehension is cleaner